libraries/rush-lib/src/schemas/experiments.schema.json

```diff
@@ -66,6 +66,10 @@
       "description": "If true, when running in watch mode, Rush will check for phase scripts named `_phase:<name>:ipc` and run them instead of `_phase:<name>` if they exist. The created child process will be provided with an IPC channel and expected to persist across invocations.",
       "type": "boolean"
     },
+    "allowCobuildWithoutCache": {
```

Comment:
Add this to https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/assets/rush-init/common/config/rush/experiments.json.

Reply:
updated in [b2e50c6](https://github.com/microsoft/rushstack/pull/4871/commits/b2e50c6c935557d17309ad79a97c6bc8724f7d40)